integration-tests/src/test/resources/model-in-image/model.jdbc.properties

```diff
@@ -0,0 +1,9 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+
+MANSVR_PORT=8001
+dbcs1.url=slcai724.us.oracle.com:1583:w03ys12c
```

Comment:
no oracle internal system names to be checked into public repositories

Reply:
oh my word. I did this. Sorry. Was for testing purpose. forgot to change. Good Catch!